app/views/directives/edit-environment-from.html

```diff
@@ -23,8 +48,13 @@
           </div>
           <div ng-if="entry.configMapRef.name || entry.secretRef.name" class="faux-form-control readonly">
             Use all keys and values from
-            <span ng-if="entry.configMapRef.name">config map {{entry.configMapRef.name}}</span>
-            <span ng-if="entry.secretRef.name">secret {{entry.secretRef.name}}</span>
+            <span ng-if="entry.prefix">prefixed</span>
+            <span ng-if="entry.configMapRef.name">config map
+              <span ng-if="entry.prefix">"{{entry.prefix}}"</span>
+              {{entry.configMapRef.name}}</span>
+            <span ng-if="entry.secretRef.name">secret
+              <span ng-if="entry.prefix">"{{entry.prefix}}"</span>
+              {{entry.secretRef.name}}</span>
```

Comment:
Maybe...?

```
Use all keys and values from config map my-config-map, adding name prefix "my-prefix-"
```

@jwforres is better at wording stuff then me, though.

Reply:
tough to stuff that into one sentence, would probably read better as two.

"Use all keys and values from config map my-config-map. Names will be prefixed with "my-prefix".